Makefile

```diff
@@ -1,2 +1,6 @@
-flake: 
+test-quick:
+	env SKIP_MIGRATIONS=1 ./manage.py test -s --nologcapture
```

Comment:
#159 introduce new command - ``test_dnsaas`` use this instead of ``./manage.py``